pkg/handler/collector/github/github.go

```diff
@@ -227,6 +278,58 @@ func (g *githubCollector) collectAssetsForRelease(ctx context.Context, release c
 	}
 }
 
+// fetchWorkflowRunArtifacts fetches the artifacts from the GitHub Action Workflow runs for a given owner and repo.
+// The artifacts are then sent to the provided docChannel.
+// If an error occurs while fetching a workflow run or its artifacts, it is logged and the function continues to the next workflow run.
+func (g *githubCollector) fetchWorkflowRunArtifacts(ctx context.Context, owner string, repo string, docChannel chan<- *processor.Document) {
```

Comment:
Because `fetchWorkflowRunArtifacts` is a method on `githubCollector`, it doesn't need the `owner` and `repo` parameters. Just access those in the `githubCollector` object.

Reply:
Thanks, I didn’t see that!